robocop/checkers/comments.py

```diff
@@ -7,21 +7,31 @@
 from robot.utils import FileReader
 
 from robocop.checkers import RawFileChecker, VisitorChecker
-from robocop.rules import Rule, RuleSeverity
+from robocop.rules import Rule, RuleSeverity, RuleParam
 from robocop.utils import ROBOT_VERSION
 
 rules = {
     "0701": Rule(
+        RuleParam(
+            name="todos",
+            default="todo,fixme",
+            converter=str,
+            desc="List of terms that violate the rule in comments.",
+        ),
         rule_id="0701",
         name="todo-in-comment",
-        msg="Found {{ todo_or_fixme }} in comment",
+        msg="Found {{ marker }} in comment",
         severity=RuleSeverity.WARNING,
         docs="""
         Example::
```

Comment:
Maybe prepend it with sentence like: "Report occurences of the configured, case insensitive marker in the comments. By default it reports TODO and FIXME markers."

Reply:
I can't see this text in the code. @rikerfi Can you add it?

Reply:
> Maybe prepend it with sentence like: "Report occurences of the configured, case insensitive marker in the comments. By default it reports TODO and FIXME markers."

Ah, this is something we do not want to be the rule message, but the `RuleParam` description. Do you agree?

Reply:
Actually I was thinking to add it to `docs`, not to the `desc`. Description was fine, IMO.
@bhirsz Please advise.
@rikerfi Sorry for confusion.

Reply:
Yeah, it should be in the docs of the Rule (since it's general information). RuleParam desc is mostly for short information on given parameter, and it don't have to contain defalt value (it's automatically loaded to external docs anyway). It can be short like just "Marker"

Reply:
Yes, so I would revert the last change and add the mentioned sentence to the docs instead of adding it to the description.

Reply:
Oukei, to summarize. Desired `desc`, `msg` and `docs` are:

```python
desc="List of case insensitive markers that violate the rule in comments."
msg="Found a marker '{{ marker }}' in the comments"
docs="""
Report occurrences of the configured, case-insensitive marker in the comments. By default, it reports TODO and FIXME markers.

Example::
        
       # TODO: Refactor this code
       # fixme
        
Configuration example::

       robocop --configure "todo-in-comment:markers:todo,Remove me,Fix this!"

"""
```
I got it right? :smile:

Reply:
Yes :)